### PR TITLE
Use empty alt attrs for decorative elements

### PR DESCRIPTION
**Pull request by astronomersiva**

### CODE REVIEW

**Comment by astronomersiva** [comment]
Fixes #77

File: _includes/footer.html [hunk 1]
```diff
@@ -8,11 +8,11 @@
         </ul>
       </div>
       <div class="col-md-6 text-center">
-        {% asset logo-small.png alt="PyCon India 2019 Logo" width="70%" %}
+        {% asset logo-small.png alt="PyCon India 2019" width="70%" %}
       </div>
       <div class="col-md-3 text-center">
         <p class="design-by">Design by</p>
-        <a target="_blank" href="http://webchirpy.com">{% asset wclogo.png alt="Webchirpy Logo" width="40%" %}</a>
+        <a target="_blank" href="http://webchirpy.com">{% asset wclogo.png alt="Webchirpy" width="40%" %}</a>
```
Comment (bravegnu) [inline _includes/footer.html]: "Webchirpy Logo" makes more sense here.

File: _includes/footer.html [hunk 2]
```diff
@@ -8,11 +8,11 @@
         </ul>
       </div>
       <div class="col-md-6 text-center">
-        {% asset logo-small.png alt="PyCon India 2019 Logo" width="70%" %}
+        {% asset logo-small.png alt="PyCon India 2019" width="70%" %}
```
Comment (bravegnu) [inline _includes/footer.html]: If we consider this just to be an eye candy, we set it to alt=""

File: _includes/footer.html (hunk 1 above)
Comment (astronomersiva) [reply _includes/footer.html]: Fixed

File: _includes/footer.html (hunk 2 above)
Comment (astronomersiva) [reply _includes/footer.html]: I don't think the footer can be considered as eye candy.

**Comment by bravegnu** [comment]
Can we remove the alt here: https://github.com/pythonindia/inpycon2019/blob/ddbd7f4af58703c45c2627b9e4681a7836b5c481/_includes/sponsors.html#L109

**Comment by astronomersiva** [comment]
Take a look @bravegnu

**Review by bravegnu — APPROVED** [review]

**Pull request merged**